.github/workflows/tests.yml

```
@@ -41,17 +65,25 @@ jobs:
           - eigen
           - fftw
       setenv: |
+        CRDS_PATH: /tmp/data/crds_cache/
+        CRDS_SERVER_URL: ${{ github.event_name == 'workflow_dispatch' && inputs.crds_server || 'https://roman-crds.stsci.edu' }}
+        CRDS_CONTEXT: ${{ needs.crds_context.outputs.context }}
+        CRDS_CLIENT_RETRY_COUNT: 3
+        CRDS_CLIENT_RETRY_DELAY_SECONDS: 20
         STPSF_PATH: ${{ needs.latest_data_cache.outputs.cache_path }}/stpsf-data/
         GALSIM_CAT_PATH: ${{ needs.latest_data_cache.outputs.cache_path }}/galsim_data/real_galaxy_catalog_23.5_example.fits
         FFTW_DIR: /opt/homebrew/opt/fftw/lib/
       cache-path: ${{ needs.latest_data_cache.outputs.cache_path }}
-      cache-key: ${{ needs.latest_data_cache.outputs.cache_key }}
+      cache-restore-keys: |
+        ${{ needs.latest_data_cache.outputs.cache_key }}
+      cache-key: ${{ needs.latest_data_cache.outputs.cache_key }}-${{ needs.crds_context.outputs.context }}
       envs: |
         - linux: py311-oldestdeps-cov-xdist
         - linux: py311-xdist
         - linux: py312-xdist
+        - linux: py313-xdist
         # `tox` does not currently respect `requires-python` versions when creating testing environments;
         # if this breaks, add an upper pin to `requires-python` and revert this py3 to the latest working version
-        - linux: py3-cov-xdist
+        - linux: py313-cov-xdist
           coverage: codecov
-        - macos: py3-xdist
+        - macos: py313-xdist
```

remove the `-xdist`s to attempt to solve CRDS sync issues
```suggestion
       - linux: py311-oldestdeps-cov
       - linux: py311
       - linux: py312
        - linux: py313
       # `tox` does not currently respect `requires-python` versions when creating testing environments;
       # if this breaks, add an upper pin to `requires-python` and revert this py3 to the latest working version
        - linux: py313-cov
         coverage: codecov
        - macos: py313
```